### ISSUE TITLE
Replace / tweak vprojtomake so we can ditch 2003 sln/projs

### ISSUE THREAD

**Issue by mikeparker** [fortressforever/fortressforever#16]
Life would be easier if everyone (including linux) just used the 2005 files. This should be fairly easy, we just need to upgrade valves vprojtomake util so it reads 2005 instead of 2003 files.

The old one is here
https://github.com/fortressforever/fortressforever/tree/beta/utils/vprojtomake
The old instructions are here
https://developer.valvesoftware.com/wiki/Compiling_under_Linux
And the new 2010 (backwards compatible) version is here
https://developer.valvesoftware.com/wiki/Vprojtomake_2010

Apparently valve have (accidentally?) removed the hlds.so linux so we can only compile under windows right now. Squeek mentioned he might try to talk to a NeoTokyo dev to see what they did.

**Comment by DexterHaslem** [comment]
doh i just made a duplicate of this

**Comment by DexterHaslem** [comment]
this is blocked by #17 as the makefile generator stuff needs tier0 & vstdlib to build

**Comment by squeek502** [comment]
Closing this. Reopen if #17 ever gets resolved.

**Comment by squeek502** [comment]
Potentially possible after getting the pre-steampipe srcds binaries from psychonic.
